README.md

```diff
@@ -77,13 +77,147 @@ on VA.gov, you'll need to build these static pages using the following commands:
+
+<details>
+
+<summary>Example of a Drupal entity processed throughout a layout file</summary>
+
+```json
+{
+  "entityBundle": "landing_page",
+  "entityId": "79",
+  "entityPublished": true,
+  "title": "VA records",
+  "entityUrl": {
+    "breadcrumb": [
+      {
+        "url": {
+          "path": "/",
+          "routed": true
+        },
+        "text": "Home"
+      },
+      {
+        "url": {
+          "path": "",
+          "routed": true
+        },
+        "text": "Records"
+      }
+    ],
+    "path": "/records"
+  },
+  "fieldIntroText": "Access your VA records and documents online to more easily manage your benefits."
+}
+```
+
+Whereas Markdown files contain a clear `layout` declaration in frontmatter, the `entityBundle` is used to form the relationship between a CMS data structure to a layout. The `entityBundle` can be considered a the type of CMS "node" represented by this data structure. Layouts used solely by CMS data structures have a unique file extension - `.drupal.liquid`. In this example, the layout file would be `vets-website/src/site/layouts/landing_page.drupal.liquid`.
+
+```
+<h1>{{ title }}</h1>
+<p>{{ fieldIntroText }}</p>
+```
+
+This layout would result in this HTML snippet -
+
+```html
+<title>VA records</title>
+<p>Access your VA records and documents online to more easily manage your benefits.</p>
+```
+
+The `entityUrl.path` property is used to determine the page's path on the website. This example would be visible at `/records/`.
+
+</details>
+
+#### Debugging
+
+<details>
+
+<summary>Console output</summary>
+
+While running your website locally, open the console panel of your browser's developer tools. Visit `/health-care` and observe the following output.
+
+![console output of Drupal data](.github/images/cms-data-console.png)
+
+This data structure represents the data structure that was processed throughout the layout file during the website compilation.
+</details>
+
+<details>
+
+<summary>DOM attributes</summary>
+To further demystify the dynamic nature of the templating process, a pattern of the project is to use `data-` attributes to describe various templates. For example, observe the various `data-template` properties in the following DOM.
+
+![data attributes describing the templating process](.github/images/cms-data-attributes.png)
+
+There is also an HTML comment at the top of the file to help describe the current page.
+
+![HTML comment describing the page](.github/images/cms-html-comment.png)
+
+</details>
+
+#### Resources
+
+The implementation of the Liquid templating language used by our project is [TinyLiquid](https://github.com/leizongmin/tinyliquid). I highly recommend using [TinyLiquid's README](https://github.com/leizongmin/tinyliquid/blob/cebcb26c3839e725cf0469dccc0073799902a020/README_en.md) as your resource while writing a template, as there are small [differences from Shopify's language spec](https://github.com/leizongmin/tinyliquid#the-difference-with-liquid-language).
```

Comment:
I opened this PR solely to get a link to the TinyLiquid filter spec into our README.

Reply:
hahaha backlinks for dayz